counsel.el

```diff
@@ -1812,11 +1812,25 @@ currently checked out."
     (define-key map [remap undo] 'counsel-find-file-undo)
     map))
 
+;;* File
+;;** `counsel-file-jump'
+(defvar counsel-file-jump-map
+  (let ((map (make-sparse-keymap)))
+    (define-key map (kbd "`") #'counsel-file-find-from-jump)
+    map))
```

Comment:
Nit: Please always add docstrings. (I know a lot of existing code lacks them, but let's not proliferate that.)